.github/workflows/go.yml

```diff
@@ -30,24 +30,27 @@ jobs:
           fetch-depth: 0
 
       - name: Start pulling environment in the background
-        run: make env-pull &
+        run: ./bin/task env-pull &
```

Comment:
Not for this PR, but can task run targets in parallel?

Reply:
We can do something like this, as docs said:
```
You can also make the tasks given by the command line run in parallel by using the --parallel flag (alias -p). Example: task --parallel js css.
```